frontend/public/locales/en/billings.json

```diff
@@ -0,0 +1,15 @@
+{
+	"subscription_past_due": "Your subscription status is past due",
```

Comment:
@SagarRajput-7: The message copy sounds incomplete. We need to provide additional info here.

Reply:
`Your subscription status indicates that payment is past due. Please renew your subscription to continue enjoying uninterrupted service.`

Reply:
Done, added suggested changes